dashboard/migrations/0015_auto_20200312_1323.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.10 on 2020-03-12 17:23
```

Comment:
I think it would make sense to add a comment saying you modified the auto-generated version. We should also probably change the migration file name so it doesn't say 'auto'.